travis.sh

```diff
@@ -1,27 +0,0 @@
-#!/bin/bash
```

Comment:
Same as https://github.com/ros-industrial/industrial_ci/pull/157/files#r109773432, why GitHub shows this proposed content of `travis.sh` as a new file?